src/shared/leftSidebar/leftSidebar.css

```diff
@@ -0,0 +1,67 @@
+.leftSidebar-container {
+  min-height: 88vh;
+  max-height: 88vh;
+  border-right: 1px solid var(--primary-color);
+  position: sticky;
+  top: 12vh;
+  left: 0;
+}
+.leftSidebar-list-items {
+  padding: 15% 10%;
```

Comment:
Why padding in %??

Reply:
because it should change according to screen size . Is giving in % is wrong ??